starter-code/javascript/chronometer.js

```diff
@@ -1,40 +1,68 @@
-// Constructor
-// function Chronometer() {
+//Constructor
+function Chronometer() {
+    this.currentTime = 0;
+    this.intervalId = 0;
+    this.minutes = '0';
+    this.seconds = '0';
+}
 
-// }
+var Chrono = new Chronometer();
 
-// Chronometer.prototype.startClick = function () {
+Chronometer.prototype.startClick = function() {
+    this.intervalId = setInterval(function() {
+            this.currentTime += 1
+            this.setTime();
+        }.bind(this),
+        1000);
 
-// };
+    //setTime = évol html
+};
 
-// Chronometer.prototype.setMinutes = function () {
-  
-// };
+Chronometer.prototype.setMinutes = function() {
+    var minute = parseInt(this.currentTime / 60);
+    return minute;
+};
 
-// Chronometer.prototype.setSeconds = function () {
-  
-// };
 
-// Chronometer.prototype.twoDigitsNumber = function () {
-  
-// };
+Chronometer.prototype.setSeconds = function() {
+    var second = this.currentTime % 60;
+    return second;
+};
 
-// Chronometer.prototype.setTime = function () {
 
-// };
+Chronometer.prototype.twoDigitsNumber = function(number) {
+    if (number < 10) {
+        return number = '0' + number;
+    } else {
+        return number += '';
```

Comment:
Note that since you return in your `if`, the useless keyword is useless. To make your code cleaner you could have done this:
```javascript
    if (number < 10) {
        return number = '0' + number;
    } 
    return number.toString();
```